Python/hard/hanoi/solution.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+from os import system
+import time
+import sys
+
+def solve(towers, a, b, c, i):
```

Comment:
Please use more precise variable names instead of `a, b, c`

Reply:
... and i :rofl:

Reply:
... and add a nice docstring explaining the function

Reply:
docstring fine?